api/serverless.yml

```diff
@@ -35,39 +35,39 @@ functions:
     handler: users/usersHandler.users
     events:
       - http:
-          path: users/getUsers
+          path: users
           method: get
           request: ${file(./templates.yml):request}
       - http:
-          path: users/createUser
+          path: users/user
           method: post
           request: ${file(./templates.yml):request}
       - http:
-          path: users/getInterns
-          method: post
+          path: users/status/{status}
```

Comment:
Consider changing the http path to query users with the status as `users/?status={status}` passing the status as a filter.